src/ext/command.rs

```diff
@@ -711,15 +710,16 @@ impl CommandDescriber for tokio::process::Command {
     }
 }
 
-// Double derefs here because this is implemented on `&T`.
 // Desugared, `&self` on `&T` means `&&T`.
-// We want to deref all the way down to `T`, which means dereferencing twice.
+// We want to dereference using *self as opposed to self.deref()
+// This is because self.deref() will cause warnings about double references (&&T)
+// Whereas *self will not cause warnings as it is more explicit
```

Comment:
```suggestion
// Dereference using `*self` as opposed to `self.deref()`.
// This is because `self.deref()` causes warnings about double references (`&&T`),
// whereas `*self` does not since it is more explicit.
```

Reminder: well formed sentences, formatting. Also, in technical writing prefer using present tense ("causes warnings", not "will cause warnings") because future tense introduces a potential point of confusion ("will cause ... when?").